prometheus-collector/MAC-3P-Docs/ConfigureRules.md

```diff
@@ -69,18 +69,20 @@ Save the json below as samplePromRuleGroup.json for the purpose of this walkthro
                 "interval": "PT1M",
                 "rules": [
                     {
-                        "record": "job_type:billing_jobs_duration_seconds:99p5m",
-                        "expression": "histogram_quantile(0.99, sum(rate(jobs_duration_seconds_bucket{service=\"billing-processing\"}[5m])) by (job_type))"
+                        "record": "instance:node_cpu_utilisation:rate5m",
+                        "expression": "expr": "1 - avg without (cpu) (sum without (mode)(rate(node_cpu_seconds_total{job=\"node\", mode=~\"idle|iowait|steal\"}[5m])))"
```

Comment:
The "expr" was causing issues for me. I was able to run the deployment once I removed it